.lintstagedrc

```diff
@@ -1,3 +1,6 @@
 {
-  "*.{none}": ["eslint --fix", "prettier --write"]
```

Comment:
we should at some point change `none` to `ts,js`, etc and fix the errors 😅  Let's decide when can we do this!

Reply:
let's do it, the sooner the better

Reply:
let's get the current PRs merged then I'll open a new one with the fixes 👀